tools/README.md

```diff
@@ -112,115 +77,199 @@ export repo_owner=<your github account>
+    For our configuration groovy scripts, we need to allow certain jenkins and third party plugin scripts to run.  We override the restrictions here.
 
-#### Admin Password
+* **init.groovy.d/003-create-jobs.groovy**
 
-The SonarQube server instance is created with standard insecure credentials (User: `admin`, PW:). This should be reset to something stronger and stored in an OpenShift secret so authorized developers can find it.
+    This groovy script will build 2 jobs in Jenkins.  One that will build the master branch, and one that will build pull requests.
 
-The BCDevOps SonarQube repo provides a script that will generate a random PW, set it in SonarQube, and create a secret. This can be found under the  `/provisioning` folder of the cloned BCDevOps repo.
+    To add or change the jobs, this is where you want to go.  The name of this file is important, as it needs to get run *BEFORE* the 003-register-github-webhooks.groovy included in the basic install.  Scripts are run alphabetically.  The jobs need to be created before the github webhooks are created.  Our jobs script will read secrets and configmaps created during this setup; described below.
 
-In order to directly get the password reset script, run the following:
+    These jobs are configured to use [Jenkinsfile](https://github.com/bcgov/nr-showcase-devops-tools/blob/master/Jenkinsfile) and [Jenkinsfile.cicd](https://github.com/bcgov/nr-showcase-devops-tools/blob/master/Jenkinsfile.cicd) found at the root of your project repository.  These Jenkinsfiles will make use of the OpenShift ConfigMaps we will create below.
+
+* **init.groovy.d/100-jenkins-config-set-admin-address.groovy**
+
+    This groovy script will update the admin email address.  It is not necessary, but an example of further customization to Jenkins.  If you are swiping this, keep in mind the email address is hardcoded.
+
+### Project Prerequisites
+
+We are setting up a CICD pipeline to build and deploy a specific project.  In order for this to work, there are expectations on those projects.  Currently, each project will require 3 files at the __root__ of the repository.
+
+| Name | Description |
+| --- | --- |
+| [Jenkinsfile](https://github.com/bcgov/nr-showcase-devops-tools/blob/master/Jenkinsfile) | this is the master branch build pipeline |
+| [Jenkinsfile.cicd](https://github.com/bcgov/nr-showcase-devops-tools/blob/master/Jenkinsfile.cicd) | this is the Pull Request build pipeline |
+| [sonar-project.properties](https://github.com/bcgov/nr-showcase-devops-tools/blob/master/sonar-project.properties) | this is will configure your SonarQube scans |
+
+This repository contains skeleton examples, see projects [NR Get Token](https://github.com/bcgov/nr-get-token.git) and [NR Messaging Service Showcase](https://github.com/bcgov/nr-messaging-service-showcase.git) for full examples.
```

Comment:
Maybe remove This repository and say For futher examples, see... and move this paragraph one down.

Reply:
This area was just reordered.